docs/data/migration/migration-pickers-v6/migration-pickers-v6.md

```diff
@@ -64,6 +66,40 @@ For example:
 The same applies to `slotProps` and `componentsProps`.
 :::
 
+### Slot `shortcuts`: add new parameters to the `onChange` callback
```

Comment:
WDYT about aligning the section heading to be more similar to the other existing sections? 🤔 
```suggestion
### Add new parameters to the `shortcuts` slot `onChange` callback
```